### ISSUE TITLE
not skipping tests

### ISSUE THREAD

**Issue by maelle** [ropensci/onboarding-meta#59]
maybe rather an issue for one of the support packages but putting this here in order not to forget: the editor or the reviewer(s) potentially with the help of an automatic tool, should check whether the use of `testthat` skip functions are justified, in order to avoid e.g. having a green Appveyor badge with all tests skipped on Windows, when tests could actually be run on Windows.

**Comment by sckott** [comment]
good idea, i don't think we need to check skip_on_cran, but appveyor/travis makes sense

**Comment by maelle** [comment]
yeah `skip_on_os` and friends

**Comment by maelle** [comment]
#63 would handle this

**Comment by maelle** [comment]
@mpadge is this of any interest for pkgreport?

**Comment by mpadge** [comment]
Too difficult to automate, because there are so many ways tests can be skipped, and the only possible path to automation would currently be to run full `covr` in a couple of attempts at environmental replication (CRAN, not-CRAN), but that would minimally require two full install-compile-test runs, which for general usage would simply be too time consuming. And but then lots of packages have all sorts of other test flags (like `tidyverse`-style ones with extended tests switched on only for odd-number pkg versions.) If this is to be considered, I would strongly suggest that the only sensible way would be to add an explicit checklist item for reviewers to consider. That said, once we switch away from `goodpractice` to running our own custom `covr` call, we'll have full coverage reports from all aspects of package, so I don't think skipping or otherwise will be an issue.
